mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    function __construct()
+    {
+
+    }
+
+    public function getCSVHeaderForStatsExp($experiment)
+    {
+        if (strcmp($experiment, "ex1") === 0) {
+            return "detector,project,synthetics,misuses,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        } elseif (strcmp($experiment, "ex2") === 0) {
+            return "detector,project,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        } else {
+            return "detector,project,misuses,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        }
+    }
+
+    public function createCSVFromStats($experiment, $stats)
+    {
+        $result = "sep=,\n" . $this->getCSVHeaderForStatsExp($experiment);
+        foreach ($stats as $key => $stat) {
+            $result .="\n";
```

Comment:
I would prefer if we added lines to an array and do an `implode('\n', $lines)` at the end. This way we cannot forget a line break.